src/main/scala/org/deepdive/inference/SQLInferenceRunner.scala

```diff
@@ -547,11 +547,7 @@ trait SQLInferenceRunner extends InferenceRunner with Logging {
                 |= t1.${dataStore.quoteColumn(s"${InferenceNamespace.getBaseTableName(v.relation)}.id")}
                 |""".stripMargin.replaceAll("\n", " ")).mkString("AND")
             val weightJoinlist = factorDesc.weight.variables.map(v => {
-              // split column to get relation name
-              val colSplit = v.split('.')
-              val lastvrel = InferenceNamespace.getBaseTableName(colSplit(0))
-              val lastv = s"${lastvrel}.${colSplit.takeRight(colSplit.length-1).mkString(".")}"
-              s""" t0.${dataStore.quoteColumn(v)} = t1.${dataStore.quoteColumn(lastv)}"""
+              s""" t0.${dataStore.quoteColumn(v)} = t1.${dataStore.quoteColumn(v)}"""
```

Comment:
The grounding code matches ddlog.jar within deepdive.

Previously, in the ddlog generated application.conf, we name weight column as "relation.relationAlias.columnName", where for incremental case relation needs to be prefixed with dd_delta_ or dd_new. This is very awkward.

Now, we use consistent naming for weight column. The name for weight column would be "dd_weight_column_idx", and is consistent for increnental or non-incremental run.